test/fixtures/dialects/tsql/goto_statement.sql

```diff
@@ -1 +1,3 @@
 GOTO Branch_Three;
+Branch_Three:
+    RETURN
```

Comment:
Is this related to the rest of the PR?

Reply:
`Branch_Three` is the label. And then need a command after it.

#3147 had two issues:
- Label
- `GRANT`

Probably should really be two PRs but since it was one issue I fixed both in this PR (and also added `DENY` and `REVOKE` - which also fixed #1780).